lib/Model/DataManagementPlan/DataManagementPlan.hs

```diff
@@ -24,3 +29,37 @@ instance Eq DataManagementPlan where
     _dataManagementPlanUuid a == _dataManagementPlanUuid b &&
     _dataManagementPlanQuestionnaireUuid a == _dataManagementPlanQuestionnaireUuid b &&
     _dataManagementPlanFilledKnowledgeModel a == _dataManagementPlanFilledKnowledgeModel b
+
+formatExtension :: DataManagementPlanFormat -> String
+formatExtension JSON      = ".json"
+formatExtension HTML      = ".html"
+formatExtension LaTeX     = ".tex"
+formatExtension Markdown  = ".md"
+formatExtension Docx      = ".docx"
+formatExtension ODT       = ".odt"
+formatExtension PDF       = ".pdf"
+--formatExtension RTF       = ".rtf"
+--formatExtension RST       = ".rst"
+--formatExtension AsciiDoc  = ".adoc"
+--formatExtension DokuWiki  = ".dw.txt"
+--formatExtension MediaWiki = ".mw.txt"
+--formatExtension EPUB2     = ".epub"
+--formatExtension EPUB3     = ".epub"
+formatExtension _         = ""
+
+stringToFormat :: String -> Maybe DataManagementPlanFormat
```

Comment:
Maybe we can implement `Read` Typeclass?

Reply:
I would keep it this way for now... the easiest would be to send/receive the same string with lower/upper case as is data constructor and we could use derived `Read`, but this might be the most flexible for now.

Reply:
That would make sense. Could you please to change to auto derivation of `Read` and inform @janslifka that he has to implement the change on client side. And please update API docs if necessary.

Reply:
Case insensitive instance of `Read` done in https://github.com/DataStewardshipWizard/dsw-server/pull/6/commits/796aa12bf10c28f6bd19b8c225a5ed857f3bce9e